packages/cinterop/src/commonMain/kotlin/io/realm/internal/interop/sync/SyncErrorCodeCategory.kt

```diff
@@ -0,0 +1,9 @@
+package io.realm.internal.interop.sync
+
+expect enum class SyncErrorCodeCategory {
```

Comment:
Where are we indicating the recoverability aspect of the error?

Reply:
We don't right now. See https://github.com/realm/realm-kotlin/pull/762#discussion_r845405228